### DISCUSSION TITLE
Do we need feature "Eclipse Core Runtime Infrastructure"?

### DISCUSSION THREAD

**Discussion by vogella** [eclipse-platform/eclipse.platform.releng.aggregator#1527] [Q&A]
I noted today a Infrastructure feature wrapping only the equinox core feature?

![image](https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/assets/139910/7c703f03-fd7d-461c-b9de-7a8a4bb04654)

Do we use this? cc @merks 

IIRC @HannesWell did recent a clean-up of unused products. Should we also do this for unused features if they exist?

**Comment by akurtakov** [comment]
@tjwatson WDYT?

**Comment by merks** [comment]
It does a bit more than include a feature:

![image](https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/assets/208716/0ea0c26a-7132-4c68-906a-cac389544c0c)

The question of do "we" need it depends on who "we" are.   None of the Eclipse project projects need it.

**Comment by basilevs** [comment]
RCPTT and three proprietary Eclipse products I've worked on used org.eclipse.core.runtime and have not considered org.eclipse.equinox.core.
